examples/terraform-submodules/gke/README.md

```diff
@@ -0,0 +1,33 @@
+This terraform module creates an Agones cluster on GKE. 
+
+# Prerequisites
+- Terraform 0.12.x
+- Helm 3.x
+- An existing GCP Project
+- Enabled service API(s)
+
+# Enabling Service APIs
+
+Enable each required API below by following [the enabling APIs guide](https://cloud.google.com/endpoints/docs/openapi/enable-api#gcloud).
+
+Required APIs:
+- container.googleapis.com (Kubernetes)
+
+# Examples
+
+## Create an Agones in a default VPC.
+
+
+Required fields:
+- project
+
+```terraform apply -var project="<YOUR_GCP_ProjectID>" [-var agones_version="1.6.0"]```
+
+## Create an Agones cluster in a custom VPC.
+Required fields:
+- project
+- network
```

Comment:
Can we make it so that the default value for network & subnetwork is the default one? 

So people don't have to specify it if they don't want to worry about networking?

Reply:
Hi @markmandel it already does. 

## Create an Agones in a default VPC.

Required fields:
- project

Reply:
OOOH! I didn't read it correctly. Got it. Thanks! :+1: 

So, the only other comment I would make is - all these docs, should be part of https://agones.dev/site/docs/installation/terraform/gke/

Not as a README in this directory :smile: 

Have a look at https://agones.dev/site/docs/contribute/ on how to use feature shortcodes so we can hide the new functionality until the next release.

Reply:
Sg! I will try to find time today for this change.

Reply:
>So, the only other comment I would make is - all these docs, should be part of https://agones.dev/site/docs/installation/terraform/gke/
Agree, that is where we keep all the docs which touch users.